_episodes_rmd/03-day2.Rmd

```diff
@@ -1,82 +1,86 @@
 ---
 
-
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE)
-solutions <- FALSE
+source("../bin/chunk-options.R")
+knitr_fig_path("03-")
 ```
 
-##Load in and explore the maximum and minimum temperature rasters for Wisconsin.
-
-```{r, echo=solutions, include=solutions}
-library(raster)
-library(ggplot2)
-library(rgdal)
-```
 
+## 1) Load in the maximum and minimum temperature rasters  for Wisconsin using the `raster()` function and explore using the `summary()` function.
 
 
-```{r, echo=solutions, include=solutions}
-mintemp <- raster("../data/mintemp_monthcold_wi.tif")
-summary(mintemp)
-```
-
 
-```{r, echo=solutions, include=solutions}
-maxtemp <- raster("../data/maxtemp_monthwarm_wi.tif")
-summary(maxtemp)
-```
+> ## Solution
+> ```{r, eval=TRUE}
+> library(raster)
+> library(ggplot2)
+> library(rgdal)
+> mintemp <- raster("../data/mintemp_monthcold_wi.tif")
+> summary(mintemp)
+> maxtemp <- raster("../data/maxtemp_monthwarm_wi.tif")
+> summary(maxtemp)
+> ```
+{: .solution}
 
 
-##Are the coordinate reference systems the same in these two rasters?
 
-```{r, echo=solutions, include=solutions}
-mintemp
-maxtemp
 
-#or
+## 2) Use the `crs()` function to check if the coordinate reference systems the same in these two rasters?
 
-crs(mintemp)
-crs(maxtemp)
-```
 
-Yes, the CRS for both rasters is the same. Both are in longlat projection, datum WGS84.
+> ## Solution
+> ```{r, eval=TRUE}
+> mintemp
+> maxtemp
+> ```
+> OR
+> 
+> ```{r, eval=TRUE}
+> crs(mintemp)
+> crs(maxtemp)
+> ```
+> 
+> Yes, the CRS for both rasters is the same. Both are in longlat projection, datum WGS84.
+{: .solution}
 
-##Create visualizations of both minimum and maximum temperature rasters using `ggplot2`
 
-```{r, echo=solutions, include=solutions}
-mintemp_df <- as.data.frame(mintemp, xy=TRUE)
-maxtemp_df <- as.data.frame(maxtemp, xy=TRUE)
 
-ggplot() +
-    geom_raster(data = mintemp_df , aes(x = x, y = y, fill = mintemp_monthcold_wi)) +
-    scale_fill_viridis_c() +
-    coord_quickmap()
-ggsave("03-day2-mintempraster.png", path="../fig/")
+## 3) Create visualizations of both minimum and maximum temperature rasters using the `ggplot2` package.
 
-ggplot() +
-    geom_raster(data = maxtemp_df , aes(x = x, y = y, fill = maxtemp_monthwarm_wi)) +
-    scale_fill_viridis_c() +
-    coord_quickmap()
-ggsave("03-day2-maxtempraster.png", path="../fig/")
 
-```
+> ## Solution
+> ```{r, eval=TRUE}
+> mintemp_df <- as.data.frame(mintemp, xy=TRUE)
+> maxtemp_df <- as.data.frame(maxtemp, xy=TRUE)
+> 
+> ggplot() +
+>    geom_raster(data = mintemp_df , aes(x = x, y = y, fill = mintemp_monthcold_wi)) +
+>    scale_fill_viridis_c() +
+>    coord_quickmap()
+>
+> ggplot() +
+>    geom_raster(data = maxtemp_df , aes(x = x, y = y, fill = maxtemp_monthwarm_wi)) +
+>    scale_fill_viridis_c() +
+>    coord_quickmap()

```

Comment:
I think you might want to split up the ggplot code chunks here.  I'm betting it will better disply which plot goes with which code chunk if they are separate code chunks.